akka-actor-typed/src/main/scala/akka/actor/typed/internal/ExtensionsImpl.scala

```diff
@@ -25,7 +25,7 @@ trait ExtensionsImpl extends Extensions { self: ActorSystem[_] ⇒
   /**
    * Hook for ActorSystem to load extensions on startup
    */
-  protected final def loadExtensions(): Unit = {
+  private[akka] def loadExtensions(): Unit = {
```

Comment:
annotations please

Reply:
And comment -- Java people won't see that it's private at all, it looks like a public method to them

Reply:
done